### PR TITLE
LEP-322 disassociate properties from capital_summary

### PR DESCRIPTION
**Pull request by ripan**
https://dsdmoj.atlassian.net/browse/LEP-322

Remove **properties** tables from database

### CODE REVIEW

**Comment by davidread** [comment]
DON'T MERGE until LEP-341 is merge

File: app/controllers/v6/assessments_controller.rb
Comment (patrick-laa) [inline app/controllers/v6/assessments_controller.rb:87]: Reminder to delete this commented-out code before merging.

File: app/controllers/v6/assessments_controller.rb
Comment (patrick-laa) [inline app/controllers/v6/assessments_controller.rb:119]: It would maybe be nicer if the Property object was the thing that knew about default values, rather than the AssessmentController having to be explicit about what values to use if they're not explicitly provided.

File: app/services/assessors/property_assessor.rb
Comment (patrick-laa) [inline app/services/assessors/property_assessor.rb:53]: This makes me a little anxious: Why are we adding rounding to this specific figure as part of this ticket?

File: app/models/capital_summary.rb
Comment (patrick-laa) [inline app/models/capital_summary.rb:12]: I'm guessing these two `has_many` lines are being cleared away because they're redundant and unused? They don't seem to be directly related to properties, but it's fair enough if this is just some spring cleaning as we go.

**Review by patrick-laa — APPROVED** [review]
Code looks fine to me. There's one `.round` been introduced that I'm not sure about - I wonder if maybe that's a sign that some numbers were being cast to decimals when they were persisted to the DB, and now that they're not they're maybe being handled as floats or something? Might be worth explicitly casting params to the right data type in the Property model so that it continues to behave similarly to how it did when it was tied to the database.

File: app/models/capital_summary.rb
Comment (ripan) [reply app/models/capital_summary.rb:12]: Good Catch. I will do cleanup in separate ticket.

File: app/services/assessors/property_assessor.rb
Comment (ripan) [reply app/services/assessors/property_assessor.rb]: Type casting done in Property model itself so no rounding required here

**Pull request merged**
